dpctl/memory/_memory.pyx

```diff
@@ -34,6 +34,7 @@ from cpython.bytes cimport PyBytes_AS_STRING, PyBytes_FromStringAndSize
 from cpython cimport pycapsule
 
 import numpy as np
```

Comment:
Should this be `cimport numpy as cnp`?

Reply:
No, this file only uses Python API of NumPy.

Reply:
makes sense.